solutionktlist1/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,28 @@
+fun main(args: Array<String>) {
+    var name:String?= null
+    name?.let{it
+    println("waad")}
+}
+
+    class School(){
+        lateinit var studentsNames:String
+        lateinit var teachersNames:String
+        lateinit var adminNames:String
+    }
+    fun main(){
+        var School:School?=null
+        println("School studentsNames:")
+        School?.run{
+            println(this.studentsNames)
+            var Dammam=School
+        }
+    } fun numbers(){
```

Comment:
where is the calling for this function `numbers()`

Reply:
fun number(num:Int){
          var list = mutableListOf<Int>(2,7,9,10)
        println(list)
        list.also { it
            it.add(5)
            it.remove(9)}
        println(list)}
is that right?

Reply:
> where is the calling for this function `numbers()`

fun number(num:Int){
var list = mutableListOf(2,7,9,10)
println(list)
list.also { it
it.add(5)
it.remove(9)}
println(list)}
is that right?

Reply:
I don't mean the code inside the function, You didn't call number function inside the main function, you must calling it

Reply:
fun number(num:Int){
          var number = mutableListOf<Int>(2,7,9,10)
        println(number)
        number.also { it
            it.add(5)
            it.remove(9)}
        println(number)}
I hope now its right?

Reply:
fun main(){
    }
    fun num(num:Int){

    }
        var num = mutableListOf<Int>(2, 7, 9, 10)
        println(num)
        num.also {
            it
            it.add(5)
            it.remove(9)
            println(num)
        }
    }
I hope its right now?

Reply:
fun main() {
        var School: School? = null
        println("School studentsNames:")
        School?.run {
            println(this.studentsNames)
            var Dammam = School
             numbers()
        }
    }
        fun numbers() {
            var list = mutableListOf<Int>(2, 7, 9, 10)
            println(list)
            list.also {
                it
                it.add(5)
                it.remove(9)
                println(list)
            }
        }